src/locales/tr.json

```diff
@@ -218,6 +221,9 @@
       "subtitle": "Tam resmi görmek için parçaları doğru sıraya koyun!"
     }
   },
+  "profilePage": {
+    "title": "tekrar hoşgeldin"
```

Comment:
You can capitalize first letter "Tekrar hoşgeldin".

Reply:
PTAL